manticore/__main__.py

```diff
@@ -21,7 +22,7 @@ def parse_arguments():
     parser = argparse.ArgumentParser(description='Symbolically analyze a program')
     parser.add_argument('--workspace', type=str, default=None,
                         help='A folder name for temporaries and results. (default mcore_?????)')
-    parser.add_argument('-v', action='count', default=1, help='Specify verbosity level from -v to -vvvv')
+    parser.add_argument('-v', action='count', default=0, help='Specify verbosity level from -v to -vvvv')
```

Comment:
this probably shouldn't change, we decided that the cli default should be verbosity level 1 (https://github.com/trailofbits/manticore/issues/431#issuecomment-320022337)